src/components/ShoppingList/ShoppingList.js

```diff
@@ -10,7 +10,15 @@ import { DateTime } from 'luxon';
 import ItemFilter from '../ItemFilter/ItemFilter.js';
 import ShoppingListItem from '../ShoppingListItem/ShoppingListItem.js';
 
-function ShoppingList({ listItems, loading, error, listId, handleModalOpen }) {
+function ShoppingList({
+  listItems,
+  loading,
+  error,
+  listId,
+  showDetails,
+  toggleDetailView,
```

Comment:
showDetails get passed down to ShoppingListItem. toggleDetailView can be removed if it's not needed